app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencySheet.kt

```diff
@@ -117,6 +117,7 @@ class MedicineFrequencySheet : BottomSheetActivity(), MedicineFrequencySheetUiAc
       BD -> R.id.medicineFrequencyBdRadioButton
       TDS -> R.id.medicineFrequencyTdsRadioButton
       QDS -> R.id.medicineFrequencyQdsRadioButton
+      is MedicineFrequency.Unknown -> -1 // Don't check any radio button
     }
     medicineFrequencyRadioGroup.check(id)
```

Comment:
If we pass -1 to this, it won't cause a crash or anything?

Reply:
Nope, it will return without doing anything if the id is -1 or same as already checked id